bluetooth/update/README.md

```diff
@@ -2,145 +2,133 @@
-	to_intersection, 
-	to_long, 
-	to_lat, s.geom, geo_id, s.lf_name, 
-	ST_Length(st_intersection(ST_BUFFER(ST_Transform(ST_SetSRID(line, 4326), 32190), 1.5*b.length, 'endcap=flat join=round') , ST_Transform(s.geom, 32190))) /ST_Length(ST_Transform(s.geom, 32190))
-	
-	FROM make_lines b
-
-INNER JOIN rliu.bt_centreline s 
-ON 
-ST_DWithin( ST_Transform(s.geom, 32190) ,  ST_BUFFER(ST_Transform(ST_SetSRID(line, 4326), 32190), 3*b.length, 'endcap=flat join=round') , 1000)
-
-
---AND ST_Length(st_intersection(ST_BUFFER(ST_Transform(ST_SetSRID(line, 4326), 32190), 1.5*b.length, 'endcap=flat join=round') , ST_Transform(s.geom, 32190))) /ST_Length(ST_Transform(s.geom, 32190)) > 0.4
+CREATE table mohan.bt_segments_new AS (
+WITH lookup AS (
+SELECT analysis_id, from_id, origin.int_id AS source, to_id, dest.int_id AS target
+FROM mohan.new_added_detectors 
+INNER JOIN mohan.bluetooth_nodes origin ON from_id = origin.bluetooth_id 
+INNER JOIN mohan.bluetooth_nodes dest ON to_id = dest.bluetooth_id
+),
+results AS (
+	SELECT * 
+	FROM lookup
+			 CROSS JOIN LATERAL pgr_dijkstra('SELECT id, source, target, cost FROM gis.centreline_routing_directional inner join gis.centreline on geo_id = id
+where fcode != 207001', source::int, target::int, TRUE)		 
+), 
+lines as (
+	SELECT analysis_id, street, direction, from_street, to_street, edge AS geo_id, geom 
+	FROM results			 
+INNER JOIN gis.centreline ON edge=geo_id
+INNER JOIN mohan.new_added_detectors USING (analysis_id)
+ORDER BY analysis_id
 )
-
-SELECT * 
-INTO rliu.new_bt_coflate1000 
-FROM conflation
+SELECT analysis_id, street, direction, from_street, to_street,
+	CASE WHEN geom_dir != direction THEN ST_reverse(geom) 
+	ELSE geom 
+	END AS 
+	geom
+FROM ( 
+SELECT analysis_id, street, direction, from_street, to_street, 
+		gis.twochar_direction(gis.direction_from_line(ST_linemerge(ST_union(geom)))) AS geom_dir,
+		ST_linemerge(ST_union(geom)) AS geom
+FROM lines
+GROUP BY analysis_id, street, direction, from_street, to_street) a)
 ```
 
-Matches the centreline road segment to the drawn lines. Note the buffer is really large (1000 units) to account for segments that run accross multiple roads.
-
-After proper QC (see next section) is applied, it should something like this.
+![bt_new_segments](img/new_segments.JPG)
 
-![bt_segments](img/bt_segments.PNG)
+## Things to note 
+Geostatistical lines and planning boundaries need to be avoided while pgrouting. 
 
-## Cleanup, Warnings, and Alternative Methods
+## Validating Output
+Validate the length of the segments with length ST_length(geom) and direction using gis.direction_from_line(geom) functions.If the detectors are located very close to the centerline intersections, it is not necessary to do the  centreline cutting. If any bluetooth detectors are not located at the start or end point of a centreline, we will need to cut the centreline using ST_linelocatepoint() as explained in [here.](https://github.com/CityofToronto/bdit_data-sources/issues/234) . 
```

Comment:
I think it would be helpful to include the steps to cut the line here in the readme instead of linking the issue.